integrations/terraform-cloud/.port/resources/blueprints.json

```diff
@@ -1,4 +1,64 @@
 [
+  {
+    "identifier": "terraformCloudOrganization",
+    "description": "This blueprint represents an organization in Terraform Cloud",
+    "title": "Terraform Cloud Organization",
+    "icon": "Terraform",
+    "schema": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "title": "Organization Name",
+          "description": "The name of the Terraform Cloud organization"
+        },
```

Comment:
redundant, we have default properties for each blueprint, and title and id is one of them